pkg/fanal/analyzer/analyzer.go

```diff
@@ -454,6 +454,9 @@ func (ag AnalyzerGroup) AnalyzeFile(ctx context.Context, wg *sync.WaitGroup, lim
 }
 
 func (ag AnalyzerGroup) RequiredPostAnalyzers(filePath string, info os.FileInfo) []Type {
+	if info.IsDir() {
```

Comment:
You may have failed to merge the main branch.

Reply:
what did I merge then... AYyy

Reply:
I recommend using `git merge main`. We squash and merge commits in PR. You don't have to care about linear history.